.circleci/config.yml

```diff
@@ -52,16 +52,23 @@ commands:
 jobs:
 ######################## PR OR MERGE TO STAGING ########################
   build_image:
-    <<: *default_machine_config
+    machine: # Use an image that has at least Docker 18.09 installed
+      image: ubuntu-1604:201903-01
     steps:
       - checkout
       - run:
           name: Build image
           command: |
-            docker login --username ${DOCKERHUB_USER} --password ${DOCKERHUB_PASSWORD} &&
-            export IMAGE_TAG=$([[ "$CIRCLE_BRANCH" == "staging" ]] && echo "staging-candidate" || echo "discardable") &&
-            IMAGE_NAME_CANDIDATE=snyk/kubernetes-monitor:${IMAGE_TAG}-${CIRCLE_SHA1} &&
-            ./scripts/docker/build-image.sh ${IMAGE_NAME_CANDIDATE} &&
+            docker login --username "${CONTAINER_REGISTRY_USERNAME}" --password "${CONTAINER_REGISTRY_PASSWORD}" registry.redhat.io
+            IMAGE_TAG=$([[ "$CIRCLE_BRANCH" == "staging" ]] && echo "staging-candidate" || echo "discardable")
+            IMAGE_NAME_CANDIDATE=snyk/kubernetes-monitor:${IMAGE_TAG}-${CIRCLE_SHA1}
+            RED_HAT_AUTH_CREDENTIALS_FILE=credentials
+            echo "${RED_HAT_USERNAME}" >> ${RED_HAT_AUTH_CREDENTIALS_FILE}
+            echo "${RED_HAT_PASSWORD}" >> ${RED_HAT_AUTH_CREDENTIALS_FILE}
+            # TODO: only Docker version 19.03.3 with BuildKit can pull the image using the FROM directive in the Dockerfile
+            docker pull registry.access.redhat.com/ubi8/ubi:latest
```

Comment:
are we ok with always using `latest` vs using a specific version?

Reply:
I think so, I assume we would be getting mostly security fixes with new releases and no breaking changes